### PR TITLE
[IMP] inventory: split removal strategy doc into 5

### PR DESCRIPTION
**Pull request by Felicious**
Overhaul and reorganization of removal strategies doc. Also added "least packages" section and workflow.

## Summary of changes
- Add detailed doc of the newest removal strategy, least packages, with a use case
- Consolidate "How to use" removal strategies section with the "Using removal strategies" section of the original doc. Each removal strategy has its own dedicated section now.
- Add summary of removal strategies chart at the beginning of each bulky section
- Detailed setup instructions are located in the **Configuration** section of the new version, with each removal strategy workflow using ref tags to refer back to detailed setup instructions.
- Update all screenshots to new 17.0 UI

### Updated summary of changes
- As per @hojo-odoo's suggestion, this lengthy removal strategies doc is to be split into 6 pieces:
- a summary of all the removal strategies (this one), that's intended to be published first
- 5 docs of the 5 different removal strategies:
  1. [FIFO](https://github.com/odoo/documentation/pull/7289)
  2. [LIFO](https://github.com/odoo/documentation/pull/7293/)
  3. [FEFO](https://github.com/odoo/documentation/pull/7294)
  4. [Closest location](https://github.com/odoo/documentation/pull/7295)
  5. [Least packages](https://github.com/odoo/documentation/pull/7296)

### CODE REVIEW

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 1]
```diff
@@ -123,296 +274,340 @@ lot/serial number that entered the warehouse's inventory).
...
-the lot numbers expiring first will be reserved thanks to the :abbr:`FEFO (First Expired, First
-Out)` strategy. All twenty tubes of hand cream from lot `0000001` and five from lot `0000003` will
-be selected to be sent to the customer, detailed in the :guilabel:`Detailed Operations` tab in the
-sales order.
-
-.. image:: removal/pick-hand-cream.png
+   :alt: Set least packages removal strategy on the product category.
+
+Create a :ref:`delivery order <inventory/delivery/one-step>` for eighty kilograms of flour, by
+either going to the :menuselection:`Sales app` and create a new quotation, or from the delivery
```
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:595]:
```suggestion
Create a :ref:`delivery order <inventory/delivery/one-step>` for eighty kilograms of flour,
either by going to the :menuselection:`Sales app` and creating a new quotation, or from the delivery
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 2]
```diff
@@ -123,296 +274,340 @@ lot/serial number that entered the warehouse's inventory).
...
+orders dashboard in :menuselection:`Inventory --> Operations --> Deliveries`.
+
+On the delivery order, the :guilabel:`Quantity` field displays the amount automatically picked
+according to the removal strategy. For more details about *where* the units were picked, select the
+:guilabel:`⦙≣ (bulleted list)` icon on the far right. Doing so opens the :guilabel:`Stock move`
+pop-up window that displays how the reserved items were picked according to the removal strategy.
+
+In the :guilabel:`Stock move` window, the :guilabel:`Pick from` field details where the quantities
```
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:603]:
```suggestion
:guilabel:`⦙≣ (bulleted list)` icon on the far right. Doing so opens the :guilabel:`Open: Stock move`
pop-up window that displays how the reserved items were picked according to the removal strategy.

In the :guilabel:`Open: Stock move` window, the :guilabel:`Pick from` field details where the quantities
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 3]
```diff
@@ -123,296 +274,340 @@ lot/serial number that entered the warehouse's inventory).
...
+exceeds the quantity in the opened package of `54 kg`, an unopened package of `100 kg` is selected.
+
+.. image:: removal/least-package.png
    :align: center
-   :alt: Hand cream lot numbers selected for the sales order.
+   :alt: Show which package was picked in the *Pick From* field.
+
+Doing so allows the warehouse employees to consolidate the remaining `20 kg` of flour from the `100
+kg` package with the `54 kg`, instead of picking the `54 kg` package, and picking the remaining `26
+kg` from another sealed package.
```
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:613]: I would remove this. It seems outside the scope of the removal strategy.

**Review by hojo-odoo — APPROVED** [review]
Hey @Felicious,

Nice job on this. I left a variety of comments and suggestions.

On the whole, I feel like the "Workflow" sections are a bit redundant, given that they immediately follow the example sections, which communicate more or less the same information, but in a more concise format.

Also, I feel like it would be better to have a separate doc for each removal strategy, rather than lumping them together like this.

It's up to you how to proceed regarding both of these issues though.

Thanks,
hojo

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 4]
```diff
@@ -1,119 +1,270 @@
...
-products with the closest expiration date is preferred, instead of products received first or last.
-Using the chosen removal strategy configured for those products (in this example, :ref:`FEFO
-<routes/FEFO>`), Odoo generates a transfer for the products with the soonest expiration date to the
-picking area, then the packing area, and finally, to the shipping docks for delivery to the
-customer.
-
-.. image:: removal/packing-products.png
+**Storage locations** and **multi-step routes** are necessary features for setting **all** types of
+removal strategies on a location. However, these features are specifically required for the closest
+location removal strategy since it is only applied at the location level.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:137]:
```suggestion
removal strategies on a location. 
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 5]
```diff
@@ -1,119 +1,270 @@
...
+   August 25th. For an order made on September 1st, the :abbr:`FIFO (First In, First Out)` removal
+   strategy  prioritizes lots that have been in stock the longest. So, products received on August
+   1st are selected first for picking.
+
+   .. image:: removal/fifo-example.png
+      :align: center
+      :alt: Illustration of FIFO selecting the oldest products in stock.
+
+To use the :abbr:`FIFO (First In, First Out)` removal strategy, enable the :guilabel:`Lots & Serial
+Numbers` feature and activate :guilabel:`Tracking` by lots or serial numbers on the product form.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:182]:
```suggestion

```

[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:182]:
```suggestion

```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 6]
```diff
@@ -123,296 +274,340 @@ lot/serial number that entered the warehouse's inventory).
...
+
+To view the detailed pickings, click the :guilabel:`⦙≣ (bulleted list)` icon on the far right of the
+egg's product line in the :guilabel:`Operations` tab of the delivery order. Doing so opens
+the :guilabel:`Stock move` pop-up window.
+
+In the :guilabel:`Stock move` window, the :guilabel:`Pick from` field details where the quantities
+to fulfill the :guilabel:`Demand` are picked from. Since the order demanded six cartons of eggs,
+using the :abbr:`FEFO (First Expired, First Out)` removal strategy, all five cartons from `LOT1`
+with the removal date of February 26th are picked. The remaining carton is selected from the `LOT2`,
+which has a removal date of May 11.
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:446]: Thanks for noticing that! I'll make the change (:

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst (hunk 3 above)
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:613]:
```suggestion

```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 7]
```diff
@@ -123,296 +274,340 @@ lot/serial number that entered the warehouse's inventory).
...
-   :alt: View of the lots of cinder blocks in the inventory valuation report.
+Once the locations are named in alphabetical order based on the location's proximity from the output
+or packing location, set the removal strategy on the parent location. To do that, in the
+:guilabel:`Locations` list, select the parent location of the alphabetically named storage
+locations.
 
-Lot `000020` contains three cinder blocks, lot `000030` contains five cinder blocks, and lot
-`0000400` contains four cinder blocks.
+Doing so opens the form for the parent location. In the :guilabel:`Removal Strategy` field, select
+:guilabel:`Closest Location`.
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:476]: added an internal link to a doc that explains this!

**Comment by Felicious** [comment]
Hello @odoo/crm-doc-review ! This PR is ready for your review (: Just an fyi: this doc is meant to be published with the 5 other docs listed in the summary of changes above

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 8]
```diff
@@ -1,418 +1,166 @@
...
-   :alt: Removal strategy on a product category.
+.. admonition:: Priority
 
-To view all products with lots/serial numbers assigned to them, navigate to
-:menuselection:`Inventory app --> Products --> Lots/Serial Numbers`. This reveals a page with
-drop-down menus of all products assigned lots or serial numbers, filtered by *product* by default.
-To change the category these products are filtered by, click :guilabel:`Product` (in the search bar,
-in the top right of the page) to remove the default filter, and select a new filter if desired.
+   When there are different removal strategies applied on the location and product category for a
+   product, the *force* removal strategy set on the :guilabel:`Product Category` is applied.
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:68]: Don't know if I'd do a custom admonition here when an **Important** one seems to fit nicely here.

As well, I'd be *very* specific in this section in order to minimize misunderstandings.

```suggestion
.. important::
When there are different removal strategies applied on both the location and product category for a
product, the value set on the :guilabel:`Force Removal Strategy` field on the :guilabel:`Product
Category` form is applied as top priority.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 9]
```diff
@@ -1,418 +1,166 @@
...
+   :alt: Enable lots and serial numbers.
 
-In this example, there are three lots of cinder blocks. The blocks are from the *All/Building
-Materials* category, where *FIFO* is set as the removal strategy. In the :guilabel:`Inventory
-Valuation Report`, the three different receipts are listed with the amounts.
+Next, ensure the intended product is tracked by lots or serial numbers by navigating to the product
+form through :menuselection:`Inventory --> Products --> Products`, and selecting the desired
+product. On the product form, switch to the :guilabel:`Inventory` tab, and under the
+:guilabel:`Tracking` field, select either the :guilabel:`By Unique Serial Number` or :guilabel:`By
+Lots` options.
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:120]: After selecting this on the product form, the next step is to add the lot/serial # to the product so Odoo can actually track it properly. It's not enough to just enable the setting, so this last step is what completes the instruction. 

Therefore, I'd briefly list the step(s) on how to do this (e.g. **☰ Lots/Serial Numbers** smart button on product form) or point to other documentation that has that information.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 10]
```diff
@@ -1,418 +1,166 @@
...
+   :alt: Enable the locations and route features.
 
-To see the removal strategy in action, go to the :menuselection:`Sales app` and click
-:guilabel:`Create` to create a sales order. Next, select a :guilabel:`Customer` from the drop-down
-menu. Then click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab. Select a product (for
-this example, the :guilabel:`Cinder Block`) from the drop-down menu, or type in the name of the
-product in the field. Enter a quantity (for this example, `5.00`) in the :guilabel:`Quantity` field,
-then click :guilabel:`Save`, then click :guilabel:`Confirm`.
+Expiration date
+~~~~~~~~~~~~~~~
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:138]: There are no specific navigational steps here for this setting.

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @Felicious 
See below for review items, thanks!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 11]
```diff
@@ -1,418 +1,166 @@
...
 
-Using the :abbr:`FEFO (First Expired, First Out)` removal strategy, every sales order that includes
-products with this removal strategy assigned ensures that transfers are requested for products with
-the expiration date soonest to the order date.
+.. seealso::
+   -
 
-As an example, imagine there are three lots of six-egg boxes. Those three lots have the following
-lot numbers: `20001`, `20002`, and `20003`, each with five boxes in it.
+.. _inventory/routes/strategies/set:
```
[the remark this answers is not in the document]
Comment (samueljlieber) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:43]: Hey @Felicious, I agree with @StraubCreative here. 

The custom anchor should align with the folder structure of the doc and the section you are anchoring in, so for this anchor, I would say either:
```
.. _inventory/warehouses_storage/removalConfig:
```
or
```
.. _inventory/warehouses_storage/removal-config:
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 12]
```diff
@@ -1,418 +1,166 @@
...
-products for removal based on date of entry into the warehouse, the :guilabel:`First Expired, First
-Out (FEFO)` method targets products for removal based on their assigned expiration dates.
+For comprehensive examples for how to use each removal strategy, refer to each individual
+documentation page.
 
-Using the :abbr:`FEFO (First Expired, First Out)` removal strategy, every sales order that includes
-products with this removal strategy assigned ensures that transfers are requested for products with
-the expiration date soonest to the order date.
+.. seealso::
+   -
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:41]:
```suggestion

```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst (hunk 11 above)
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:43]:
```suggestion
.. _inventory/warehouses_storage/removal-config:
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst [hunk 13]
```diff
@@ -1,418 +1,171 @@
...
+   When there are different removal strategies applied on both the location and product category for
+   a product, the value set on the :guilabel:`Force Removal Strategy` field set on a
+   :guilabel:`Product Category` form is applied as top priority.
 
-.. warning::
-   In many countries, the :abbr:`LIFO (Last In, First Out)` removal strategy in banned, since it can
-   potentially result in old, expired, or obsolete products being delivered to customers.
+   .. image:: removal/navigate-location-category.png
+      :align: center
+      :alt: Change the Force Removal Strategy for either the Product Categories or Locations.
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:68]: Was this image always set in the admonition like this or did it move since last review?
Seems to fit better right after line 44, up to you 🙂

**Review by StraubCreative — APPROVED** [review]
Looks good @Felicious 👍 
One small comment but it's optional.

On to you @samueljlieber for tech review and merge, thanks!

@robodoo delegate=samueljlieber

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious, great job on this PR. Everything looks good to me! I do agree with [ZSTs comment](https://github.com/odoo/documentation/pull/7223#discussion_r1446616110) about moving the image out of the admonition only because it seems it would flow better, but up to you if you want to remove, let me know! 

Otherwise I feel this PR is ready to merge. Thanks!

**Comment by samueljlieber** [comment]
https://github.com/odoo/documentation/commit/caeca5ff4662d17b819dbe8aeb555a4da0d3d80b looks good to me, thank you @Felicious 👍

@robodoo r+
